packages/api/src/api/expoJudgingSession/index.ts

```diff
@@ -10,3 +10,4 @@ export const expoJudgingSession = Router();
 expoJudgingSession.post('', adminMiddleware, post);
 expoJudgingSession.get('', adminMiddleware, get);
 expoJudgingSession.use('/:id', judgeMiddleware, id);
+expoJudgingSession.use('/:id', adminMiddleware, id);
```

Comment:
What's the need for this?

Reply:
The only folks that have access to results should be admins so this adds admin middleware to the route... however, this is not the right way to do it...

@Ansaar-Syed you'll need to remove `judgeMiddleware` from the line above this and have the route itself manage it's middleware one level higher up the hierarchy

Reply:
done